.github/workflows/bump_torch_mlir.yml

```diff
@@ -0,0 +1,67 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# Automatically bumps the torch-mlir submodule to match shark-infra/torch-mlir:main.
+# Manually triggered for now, but can be scheduled in the future.
+# This typically takes ~90s to run.
+name: Bump submodule - shark-infra/torch-mlir
+
+on:
+  # TODO: Uncomment the following lines to enable the schedule when this action is stable & useful enough
+  # schedule:
+  #   - cron: '0 0 * * 0,4'  # Run at 00:00 on Sunday and Thursday
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths:
+      - .github/workflows/bump_torch_mlir.yml
+
+jobs:
+  bump_submodule:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c  # v3.3.0
+        with:
+          submodules: true
+
+      - name: Setting git config
+        run: |
+          git config --local user.email "iree-github-actions-bot@google.com"
+          git config --local user.name "Submodule Bump Action"
+
+      - name: Bump torch-mlir submodule
+        id: bump_submodule
+        run: |
+          cd third_party/torch-mlir
+          git fetch origin
+          git checkout origin/main
+          TORCH_MLIR_COMMIT=$(git rev-parse HEAD)
+          cd ../..
+          git add third_party/torch-mlir
+          git commit -m "Bump torch-mlir submodule to $TORCH_MLIR_COMMIT"
+          echo "TORCH_MLIR_COMMIT=$TORCH_MLIR_COMMIT" >> $GITHUB_OUTPUT
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v6.0.5
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          branch: bump-torch-mlir-submodule
+          delete-branch: true # automatically supercedes the previous bump
+          title: 'Bump torch-mlir submodule to ${{ steps.bump_submodule.outputs.TORCH_MLIR_COMMIT }}'
+          body: |
+            This PR updates the torch-mlir submodule to https://github.com/llvm/torch-mlir/commit/${{ steps.bump_submodule.outputs.TORCH_MLIR_COMMIT }}.
+
+            If you have [the github cli](https://cli.github.com/) installed, you can check out the changes with:
+            ```
+            gh pr checkout <PR_NUMBER>
+            ```
+            to resolve conflict and fix failing test cases.
```

Comment:
(fine to ignore)

I'd personally find this message spammy. Checking out a PR is standard git/GitHub stuff - anyone should know how to do that for any PR... not much reason to reference it on each generated PR. Maybe others would find it helpful though.